generators/server/templates/quarkus/pom.xml.ejs

```diff
@@ -170,6 +170,12 @@
             <version>${camel-quarkus-jackson.version}</version>
         </dependency>
         <%_ } _%>
+        <%_ if (!skipUserManagement) { _%>
+        <dependency>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-lang3</artifactId>
```

Comment:
Why do we need commons-lang?

Reply:
If I don't add it, a CI build fails when trying to compile `RandomUtil.java` that imports it. My guess is that the Quarkus H2 dependency was pulling it in before.

Reply:
Ok. let it as is.
I'll tend to remove it later on.
I'm not a fan of pulling a full dependency for basically one class...